From a5e9876f8ae5e502d9f369f514a104812159285e Mon Sep 17 00:00:00 2001
From: Tobias Schlemmer <keinstein@users.sourceforge.net>
Date: Sat, 6 Jan 2018 22:33:07 +0100
Subject: Update .gitignore from Debian packaging branch

The Debian packaging git branch contains these updates to the upstream
.gitignore file(s).  This patch is autogenerated, to provide these
updates to users of the official Debian archive view of the package.
---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index ed06009..435b034 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+/.pc
 build/
 man/*.1
 .lock*
